x/staking/handler.go

```diff
@@ -126,6 +126,8 @@ func handleMsgCreateValidator(ctx sdk.Context, msg types.MsgCreateValidator, k k
 		return err.Result()
 	}
 
+	validator.MinSelfBond = msg.MinSelfBond
```

Comment:
Should we have an additional safety check (positive value) here?

Reply:
It's done in the ValidateBasic() of the Msg